src/helpers/sorts.ts

```diff
@@ -114,6 +114,19 @@ export function getDefaultAttributeGeoPushpinSort(afm: AFM.IAfm): AFM.SortItem[]
     return [];
 }
 
+export function getDefaultHeatmapSort(resultSpec: AFM.IResultSpec): AFM.SortItem[] {
+    if (resultSpec && resultSpec.sorts) {
+        return resultSpec.sorts;
+    }
+
+    const rowsAttributeIdentifier: string = get(resultSpec, "dimensions[0].itemIdentifiers[0]");
```

Comment:
please don't use `get` function it's cursed. You will lose type information which is essential for future refactoring. `get` was used in javascript code before typescript because there was nothing to lose.